src/markdown_parser.py

```diff
@@ -51,9 +51,22 @@ def urlreplace(matchobj: Match[str]) -> str:
 
         return re.sub(URL_REGEX, urlreplace, text)
 
+    def link(self, link, text=None, title=None):
+        is_asana_vanity_link = (
+            link.startswith("https://app.asana.com") and text is not None
+        )
+        if text is None:
+            text = link
+
+        asana_tags = 'data-asana-dynamic="false" ' if is_asana_vanity_link else ""
+        s = "<a " + asana_tags + 'href="' + self._safe_url(link) + '"'
+        if title:
+            s += ' title="' + mistune.escape_html(title) + '"'
```

Comment:
API HTML (a strict subset of HTML) does not allow the `title=` attribute. [public docs](https://developers.asana.com/docs/rich-text)